### PR TITLE
Miscellaneous dependencies and urls fixes

### PR DESCRIPTION
**Pull request by alarcher**
Feel free to merge if you approve.

### CODE REVIEW

File: components/desktop/gnome2/gnome-mount/Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # Copyright 2014 Alexander Pyhalov.  All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
```
Comment (pyhalov) [inline components/desktop/gnome2/gnome-mount/Makefile]: Don't touch it, I'll  remove it in several days.

File: components/desktop/gnome2/hamster-applet/Makefile [hunk 2]
```diff
@@ -28,11 +28,11 @@ COMPONENT_LICENSE=	GPLv3
 COMPONENT_LICENSE_FILE=	COPYING
 COMPONENT_FMRI=		desktop/time-tracking/hamster
 
-include $(WS_TOP)/make-rules/prep.mk
```
Comment (pyhalov) [inline components/desktop/gnome2/hamster-applet/Makefile]: The same here, it's going to be removed really soon

File: components/encumbered/vlc/vlc.p5m [hunk 3]
```diff
@@ -33,7 +33,6 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 <transform file -> add pkg.depend.bypass-generate libjpeg\.so\..*>
 
 depend fmri=service/opengl/ogl-select type=require
-depend fmri=pkg:/image/library/libjpeg8-turbo@1.5.1-2017.0.0.0 type=require
```
Comment (pyhalov) [inline components/encumbered/vlc/vlc.p5m:36]: Why? Does it find it now automatically?

File: components/python/pylint/Makefile [hunk 4]
```diff
@@ -75,15 +75,15 @@ COMPONENT_POST_INSTALL_ACTION += \
...
+	(cd $(PROTOUSRBINDIR); $(MV) pylint pylint-$(PYTHON_VERSION)) ;
 COMPONENT_POST_INSTALL_ACTION += \
-	(cd $(PROTO_DIR)/usr/bin; $(MV) pylint-gui pylint-gui-$(PYTHON_VERSION)) ;
+	(cd $(PROTOUSRBINDIR); $(MV) pylint-gui pylint-gui-$(PYTHON_VERSION)) ;
 COMPONENT_POST_INSTALL_ACTION += \
-	(cd $(PROTO_DIR)/usr/bin; $(MV) pyreverse pyreverse-$(PYTHON_VERSION)) ;
+	(cd $(PROTOUSRBINDIR); $(MV) pyreverse pyreverse-$(PYTHON_VERSION)) ;
 COMPONENT_POST_INSTALL_ACTION += \
-	(cd $(PROTO_DIR)/usr/bin; $(MV) symilar symilar-$(PYTHON_VERSION)) ;
+	(cd $(PROTOUSRBINDIR); $(MV) symilar symilar-$(PYTHON_VERSION)) ;
```
Comment (pyhalov) [inline components/python/pylint/Makefile:88]: What did happen to pylint sample manifest?

File: components/encumbered/vlc/vlc.p5m (hunk 3 above)
Comment (alarcher) [reply components/encumbered/vlc/vlc.p5m:36]: Yes it does, we get a duplicate action.

File: components/python/pylint/Makefile (hunk 4 above)
Comment (alarcher) [reply components/python/pylint/Makefile:88]: I guess it was not updated after python2.6 was removed. The manifest did not change.

File: components/desktop/gnome2/gnome-mount/Makefile (hunk 1 above)
Comment (alarcher) [reply components/desktop/gnome2/gnome-mount/Makefile]: OK. I can only fix it tomorrow evening.

**Comment by alarcher** [comment]
@pyhalov I removed gnome-mount and hamster-applet.

**Pull request merged**